src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -36,15 +42,19 @@
     <fields>
         <fullName>Has_Link__c</fullName>
         <defaultValue>false</defaultValue>
+        <description>The item has a link to display</description>
         <externalId>false</externalId>
         <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The item has a link to display</inlineHelpText>
         <label>Has Link</label>
         <type>Checkbox</type>
     </fields>
     <fields>
         <fullName>Image__c</fullName>
+        <description>The name of the image used in the item on the Getting Started page</description>
         <externalId>false</externalId>
         <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The name of the image is displayed</inlineHelpText>
```

Comment:
Is the name of the image, the filename, or alternate text for the image? Please be specific in the inline help text